### PR TITLE
Lots of Tweaks and changes

### PR DESCRIPTION
**Pull request by RUsami**
Added bunch of new biome spawn for alexsmobs mobs.
Flutter, Murmur, Sckreecher and Rocky Roller should spawn more frequently.
Crimson Mosquito can transform with more enemies.
Added missing item in quests.
Tweaked some mobs spawning in caves and prepared custom spawn for some mobs

### CODE REVIEW

File: overrides/config/alexsmobs.toml [hunk 1]
```diff
@@ -611,22 +611,22 @@
 		farseerSpawnRolls = 0
 		#Spawn Weight, added to a pool of other mobs for each biome. Higher number = higher chance of spawning. 0 = disable spawn
 		#Range: 0 ~ 1000
-		skreecherSpawnWeight = 10
+		skreecherSpawnWeight = 40
 		#Random roll chance to enable mob spawning. Higher number = lower chance of spawning
 		#Range: > 0
-		skreecherSpawnRolls = 1
+		skreecherSpawnRolls = 0
```
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs.toml:617]: did you test this? says range > 0 so 0 might not work

File: overrides/config/alexsmobs.toml
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs.toml:1]: changelog: increased spawn rates for cave centipede, rocky roller, flutter, jerboa, sugar glider, skreecher, murmur and beached whale

File: overrides/config/alexsmobs/alligator_snapping_turtle_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/alligator_snapping_turtle_spawns.json:1]: changelog: allow alligator, snappingturtle spawning in marsh biome

File: overrides/config/alexsmobs/anaconda_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/anaconda_spawns.json:1]: changelog: allow anaconda spawning in rainforest basin

File: overrides/config/alexsmobs/anteater_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/anteater_spawns.json:1]: changelog: allow anteater spawning in rainforest and sparse rainforest

File: overrides/config/alexsmobs/capuchin_monkey_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/capuchin_monkey_spawns.json:1]: changelog: allow monkey spawn in rainforest

File: overrides/config/alexsmobs/catfish_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/catfish_spawns.json:1]: changelog: allow catfish spawn in marsh

File: overrides/config/alexsmobs/crocodile_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/crocodile_spawns.json:1]: changelog: allow corcodile spawns in rainforest basin and marsh

File: overrides/config/alexsmobs/emu_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/emu_spawns.json:1]: changelog: remove emu spawns from savanna

File: overrides/config/alexsmobs/gorilla_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/gorilla_spawns.json:1]: changelog: add gorilla spawns in rainforest and sparse rainforest

File: overrides/config/alexsmobs/guster_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/guster_spawns.json:1]: changelog: add guster spawns in dunes biomes

File: overrides/config/alexsmobs/hummingbird_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/hummingbird_spawns.json:1]: changelog: add hummingbird spawns in rainforest and sparse rainforest

File: overrides/config/alexsmobs/jerboa_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/jerboa_spawns.json:1]: changelog: allow jervboa spawns in badlands and dunes biomes

File: overrides/config/alexsmobs/kangaroo_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/kangaroo_spawns.json:1]: changelog: remove kangaroo spawns in savanna

File: overrides/config/alexsmobs/komodo_dragon_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/komodo_dragon_spawns.json:1]: changelog: add komodo dragon spawns in pretrified dunes and sparse rainforest

File: overrides/config/alexsmobs/leafcutter_anthill_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/leafcutter_anthill_spawns.json:1]: changelog: add anthill spawns in rainforest and sparse rainforest

File: overrides/config/alexsmobs/maned_wolf_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/maned_wolf_spawns.json:1]: changelog: add maned wolf spawns to sparse jungle and rainforest

File: overrides/config/alexsmobs/rain_frog_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/rain_frog_spawns.json:1]: changelog: add rain from spawns in dunes biomes

File: overrides/config/alexsmobs/rattlesnake_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/rattlesnake_spawns.json:1]: changelog: add rattlesnake spawns in dunes biomes

File: overrides/config/alexsmobs/roadrunner_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/roadrunner_spawns.json:1]: changelog: add roadrunner spawns in dunes biomes

File: overrides/config/alexsmobs/shoebill_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/shoebill_spawns.json:1]: changelog: add shoebiull spawns in marsh

File: overrides/config/alexsmobs/tarantula_hawk_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/tarantula_hawk_spawns.json:1]: changelog: add tarantula hawk spawns in dunes biomes

File: overrides/config/alexsmobs/tasmanian_devil_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/tasmanian_devil_spawns.json:1]: changelog: add tasmanian devil spawns in wooded badlands

File: overrides/config/alexsmobs/terrapin_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/terrapin_spawns.json:1]: changelog: add terrapin spawns in marsh, swamp, mangrove spawn, and rainforest basin

File: overrides/config/alexsmobs/tiger_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/tiger_spawns.json:1]: changelog: add tiger spawns in junggle, blossom woods, rainforest

File: overrides/config/alexsmobs/toucan_spawns.json
Comment (Jonah-Hansen) [inline overrides/config/alexsmobs/toucan_spawns.json:1]: changelog: add toucan spawns in rainforest

File: overrides/config/cataclysm.toml
Comment (Jonah-Hansen) [inline overrides/config/cataclysm.toml:1]: changelog:  increase spawn rate for amethyst crabs

File: overrides/config/incontrol/spawn.json [hunk 2]
```diff
@@ -11,6 +11,63 @@
     "random": 0.85,
     "onjoin": true,
     "result": "deny"
+  },
+  {
+    "mob": ["alexsmobs:crow", "alexsmobs:crocodile", "alexsmobs:alligator_snapping_turtle", "alexsmobs:seagull", "born_in_chaos_v1:thornshell_crab", "cnb:cactem", "born_in_chaos_v1:corpse_fly", "born_in_chaos_v1:dread_hound", "born_in_chaos_v1:dire_hound_leader", "epicsamurai:kitsune", "alexsmobs:tiger", "born_in_chaos_v1:zombie_fisherman", "iter_rpg:scallop", "alexsmobs:platypus", "born_in_chaos_v1:bloody_gadfly", "cnb:little_grebe", "naturalist:snail", "endermanoverhaul:flower_fields_enderman", "endermanoverhaul:dark_oak_enderman"],
+    "maxheight": 60,
+    "result": "deny"
+  },
```
Comment (Jonah-Hansen) [inline overrides/config/incontrol/spawn.json:19]: changelog:  limit spawning to y level 60 + for these mobs

File: overrides/config/mobifier.json [hunk 3]
```diff
@@ -21163,7 +21163,7 @@
           "obscure_api:magic_resistance": "*1",
           "obscure_api:penetration": "*1",
           "obscure_api:resilience": "*1",
-          "minecraft:generic.max_health": "*1",
+          "minecraft:generic.max_health": "*2",
           "minecraft:generic.knockback_resistance": "*1",
```
Comment (Jonah-Hansen) [inline overrides/config/mobifier.json:21167]: changelog:  double polar bear health

File: overrides/config/mobifier.json [hunk 4]
```diff
@@ -15211,7 +15211,7 @@
           "obscure_api:dodge": "*1",
           "forge:step_height_addition": "*1",
           "minecraft:generic.armor": "*1",
-          "minecraft:generic.flying_speed": "*1",
+          "minecraft:generic.flying_speed": "*3",
           "dungeons_libraries:ranged_damage_multiplier": "*1",
```
Comment (Jonah-Hansen) [inline overrides/config/mobifier.json:15215]: changelog:  increase butterfly flight speed

File: overrides/config/mobifier.json [hunk 5]
```diff
@@ -21796,7 +21796,7 @@
...
```
Comment (Jonah-Hansen) [inline overrides/config/mobifier.json:21800]: changelog: increase fly flight speed

**Review by Jonah-Hansen — CHANGES_REQUESTED** [review]
in the future please do individual commits for each change.

its also good to be more specific about the changes for changelog purposes, ill highlight what I can tell in the review comments by starting it with `changelog:`, please add these to the description of this pr

File: overrides/config/alexsmobs.toml (hunk 1 above)
Comment (RUsami) [reply overrides/config/alexsmobs.toml:617]: Spawnrolls are already to 0 for many mobs like rhinoceros, sugarglider, ...

Comment (Jonah-Hansen) [reply overrides/config/alexsmobs.toml:617]: ah okay hopefully its good then!

**Review by Jonah-Hansen — APPROVED** [review]

**Pull request merged**